tests/sles4sap/hana_cluster.pm

```diff
@@ -78,6 +109,18 @@ sub run {
         sleep bmwqemu::scale_timeout(30);
         $self->do_hana_sr_register(node => $node1);
         sleep bmwqemu::scale_timeout(10);
+        if (check_var('ANGI', 'true')) {
```

Comment:
This looks almost identical to if block at L69. 
I would suggest to make it a library function as it would deduplicate a lot of code.